README.md

```diff
@@ -32,3 +32,51 @@ Out of the working groups, we anticipate recommendations on governance that will
 
 
   </details>
+
+
+## Our Playbook - Operational Guidelines
```

Comment:
@anajsana - to me, this suggests we have a playbook. But we don't presently, right?

Reply:
This was my concern as well walking through the update. I think we have pieces of this strew about but I don't believe a playbook has been developed.

Reply:
I think we've also closed the survey mentioned, correct?

Reply:
The playbook was discussed in the past to have as an extension of the Charter and for those documentation related to community processes (e.g file proposals, roles responsibilities in the community, etc). Since we are already bringing the community call roles as part of the official documentation in this repo, I thought it would be a good idea to fall under the Playbook idea, and as new processes and guidelines start to be in place, add them as new sections and have everything centralised into one big group (DRF Playbook)

Reply:
I will edit the survey info. I still need to open a new PR sharing the survey results with anonymized data for everyone to have access

Reply:
@katiewaz1977 can you please refer to the survey you mentioned? I thought I added something about the engagement survey that closed a few weeks ago, but I was wrong, so maybe you were referring to a different resource?

Reply:
The Governance section says this:

There is also a brief [survey](https://discord.com/channels/1255563562449899573/1255564574380261426/1273351313001943172) that allows people to:

Express interest in getting more involved in the Foundation formation and how
Review and give feedback on the FAQs and [challenges](https://github.com/DevRel-Foundation/governance/blob/main/challenges.md)
Express preferences for how and when the Foundation may communicate with you

It is no longer open, so this section should be removed.

Reply:
Thanks! section deleted